low-level/files/opt/seagate/sspl/prerequisites/sspl_dev_deploy.sh

```diff
@@ -0,0 +1,484 @@
+                }
+            ]
+        }
+EOF
+    # Invoke consul agent
+    CONSUL_PS=$(pgrep "consul" || true)
+    [ -z "$CONSUL_PS" ] &&
+        consul agent -dev -config-file=$CONSUL_CONFIG_FILE &>/dev/null &
+
+    TRIES=0
+    while [ -z "$CONSUL_PS" ]; do
+        sleep 3
+        TRIES=$((TRIES+1))
+        CONSUL_PS=$(pgrep "consul" || true)
+        if [ $TRIES -gt 10 ]; then
+            echo "ERROR: Consul service is not started"  2>&1 | tee -a "${LOG_FILE}"
+            break
+        fi
+    done
+    echo "Done consul setup."  2>&1 | tee -a "${LOG_FILE}"
+
+}
+
+
+insert_config_in_consul(){
+
+    # Install configobj
+    configobj_req="$(grep configobj ${SSPL_BASE_DIR}/low-level/requirements.txt)"
+    [ -n "$configobj_req" ] && python3 -m pip install $configobj_req
+
+    # Update persistent cluster_id
+    consul_cluster_id="$(consul kv get system_information/cluster_id ||:)"
+    config_cluster_id="$(sed -nr 's/^cluster_id=([^,]+)$/\1/p' $CONFIG_FILE | head -1)"
+    if [ -n "$consul_cluster_id" ]; then
+        cluster_id=$consul_cluster_id
+    elif [ -n "$config_cluster_id" ]; then
+        cluster_id=$config_cluster_id
+    else
+        cluster_id=$(uuidgen)
+    fi
+    consul kv put system_information/cluster_id "$cluster_id"
+    echo "cluster_id: $cluster_id"
+
+    # Update persistent node_id
+    consul_node_id="$(consul kv get system_information/${NODE}/node_id ||:)"
+    config_node_id="$(sed -nr 's/^node_id=([^,]+)$/\1/p' $CONFIG_FILE | head -1)"
+    if [ -n "$consul_node_id" ]; then
+        node_id=$consul_node_id
+    elif [ -n "$config_node_id" ]; then
+        node_id=$config_node_id
+    else
+        node_id=$(uuidgen)
+    fi
+    consul kv put system_information/${NODE}/node_id "$node_id"
+    echo "node_id: $node_id"
+
+    # Encrypt passwords
+    if [ -n "$RMQ_PASSWD" ]; then
+        rbmq_key=$(python3 -c 'from cortx.utils.security.cipher import Cipher; \
+            print(Cipher.generate_key('"'$cluster_id'"', "rabbitmq"))')
+        RMQ_PASSWD=$(python3 -c 'from cortx.utils.security.cipher import Cipher; \
+            print(Cipher.encrypt('$rbmq_key', '"'$RMQ_PASSWD'"'.encode()).decode("utf-8"))')
+    fi
+    if [ -n "$CNTRLR_PASSWD" ]; then
+        encl_key=$(python3 -c 'from cortx.utils.security.cipher import Cipher; \
+            print(Cipher.generate_key('"'$cluster_id'"', "storage_enclosure"))')
+        CNTRLR_PASSWD=$(python3 -c 'from cortx.utils.security.cipher import Cipher; \
+            print(Cipher.encrypt('$encl_key', '"'$CNTRLR_PASSWD'"'.encode()).decode("utf-8"))')
+    fi
+
+    # Choose sspl.conf file based on component version
+    SSPL_CONF_FILE=sspl.conf.${PRODUCT_VERSION}
+    SSPL_CONF=${SSPL_BASE_DIR}/low-level/files/opt/seagate/sspl/conf/${SSPL_CONF_FILE}
+
+    # Load sspl conf to consul
+    CONFIG_FEEDER=$SSPL_BASE_DIR/low-level/files/opt/seagate/sspl/bin/feed_sspl_conf_to_consul.py
```

Comment:
see if we can do away with this easily at this stage

Reply:
This is removed as we don't depend on consul. Now sspl.conf will have all values set with user input.